### PR TITLE
func currentWeather is showing wrong version

### PR DESCRIPTION
**Pull request by allensoberano**
In the book on chapter 12 the weather app is suppose to show the dummy data but the code provided in the start folder contains the final code with the api call. Shouldn't it be changed to the "dummy data" code that is mentioned at the start of the lesson?

### CODE REVIEW

**Review by gordonbeijing — APPROVED** [review]

**Comment by icanzilb** [comment]
Oh, nice catch! Looks good to me, what do you think @freak4pc ?

File: 12-beginning-rxcocoa/projects/starter/Wundercast/Controllers/ApiController.swift [hunk 1]
```diff
@@ -106,11 +106,15 @@ class ApiController {
 
   // MARK: - Api Calls
   func currentWeather(for city: String) -> Observable<Weather> {
-    buildRequest(pathComponent: "weather", params: [("q", city)])
-      .map { data in
-        try JSONDecoder().decode(Weather.self, from: data)
-      }
-  }
+  // Placeholder call
+  return Observable.just(
```
Comment (freak4pc) [inline 12-beginning-rxcocoa/projects/starter/Wundercast/Controllers/ApiController.swift:110]: Good catch!
We can shorten this to one of these two variations: 

```suggestion
  Observable.just(
```

```suggestion
  .just(
```

Besides, looks good. Thank you!

**Review by freak4pc — APPROVED** [review]
